cmd/wnode/main.go

```diff
@@ -76,14 +76,14 @@ var (
 
 // cmd arguments
 var (
-	bootstrapMode  = flag.Bool("standalone", false, "boostrap node: don't actively connect to peers, wait for incoming connections")
-	forwarderMode  = flag.Bool("forwarder", false, "forwarder mode: only forward messages, neither send nor decrypt messages")
+	bootstrapMode  = flag.Bool("standalone", false, "boostrap node: does not initiate connection to peers, just waits for incoming connections")
+	forwarderMode  = flag.Bool("forwarder", false, "forwarder mode: only forwards messages, neither encrypts nor decrypts messages")
```

Comment:
usually comments & CLI help messages are written in the imperative, not the infinitive. Hence, it should still be "only forward messages, neither encrypt nor decrypt ... ". An improvement is also to replace "don't initiate" with "do not initiate" as (according to my English teacher back in the day) it is associated with an instructional tone.